index.js

```diff
@@ -213,11 +214,11 @@ module.exports = {
     'spaced-comment': 0,
     'wrap-regex': 0,
     'arrow-body-style': 0,
-    'arrow-parens': 0,
+    'arrow-parens': 2,
```

Comment:
👍 

https://eslint.org/docs/rules/arrow-parens#require-parens-in-arrow-function-arguments-arrow-parens

Reply:
This is apparently a pretty contentious debate in the formatting world but I think it's better to be consistent.